draft-ietf-quic-qpack.md

```diff
@@ -179,17 +179,23 @@ x ...
 QPACK instructions occur in three locations, each of which uses a separate
 instruction space:
 
- - Table updates are carried by a unidirectional stream from encoder to decoder.
-   Instructions on this stream modify the dynamic table state without generating
-   output to any particular request.
+ - Table updates are carried by a unidirectional stream of type `0x48` from
+   encoder to decoder. Instructions on this stream modify the dynamic table
+   state without generating output to any particular request.
```

Comment:
You name this "encoder stream" in two places later, but don't establish the name here where it is defined.  "An encoder stream is a unidirectional stream that carries table updates..." and "An acknowledgment stream is a unidirectional stream that carries acknowledgments..."

Is it better as "table update stream"?

Reply:
Not specific to this PR, but happy to address separately.